controller/pagecontroller.php

```diff
@@ -40,7 +40,18 @@ public function __construct($appName, IRequest $request, $userId, $cacheManager,
 	 */
 	public function index() {
 		$params = array('user' => $this -> userId,'devices'=>$this->locationManager->loadAll($this->userId));
-		return new TemplateResponse('maps', 'main', $params);
+		$response = new TemplateResponse('maps', 'main', $params);
+		if (class_exists('OCP\AppFramework\Http\ContentSecurityPolicy')) {
+			$csp = new \OCP\AppFramework\Http\ContentSecurityPolicy();
+			// map tiles
+			$csp->addAllowedImageDomain('http://*.mqcdn.com');
+			// marker icons
+			$csp->addAllowedImageDomain('https://api.tiles.mapbox.com');
+			// images inside scripts
```

Comment:
@LukasReschke ok thanks. I just removed this line but I won't add `allowInlineScript`.